projects/1.19/assets/tanukidecor/tanukidecor/lang/zh_cn.json

```
@@ -0,0 +1,245 @@
+{
+  "itemGroup.tanukidecor": "狸猫之家",
+
+  "block.tanukidecor.alarm_clock": "闹钟",
+  "block.tanukidecor.anniversary_clock": "周年纪念钟",
+  "block.tanukidecor.antique_clock": "古董钟",
+  "block.tanukidecor.banjo_clock": "班卓琴钟",
+  "block.tanukidecor.blue_clock": "蓝色时钟",
+  "block.tanukidecor.carriage_clock": "带提手的钟",
+  "block.tanukidecor.crystal_clock": "石英钟",
+  "block.tanukidecor.cuckoo_clock": "布谷鸟时钟",

```

```suggestion
  "block.tanukidecor.cuckoo_clock": "布谷鸟钟",
```
我也搞不懂为什么官中不统一`Clock`是时钟还是钟……但是官中这么写了就这么翻吧
https://nookea.com/zh_cn/ac/3yfvreNYdjbtj84fY

---

或者全部都改成时钟来统一

---

我的建议是以动森原版为准